### DISCUSSION TITLE
Error 500

### DISCUSSION THREAD

**Discussion by RobinGooze** [firefly-iii/firefly-iii#6543] [General]
Hello, I tried to install FireFly in docker using portainer and have this issue:

![image](https://user-images.githubusercontent.com/29593419/196031680-07a7d4ed-9eb0-4977-b7db-b93006624981.png)

Before install firefly I installed mariaDB using this volumes and env's:

![image](https://user-images.githubusercontent.com/29593419/196031658-e93c2d6f-c004-4fef-b269-df0de348dc61.png)

![image](https://user-images.githubusercontent.com/29593419/196031622-2e72be43-3341-426f-b33b-90e9b4c80d96.png)

In the first time loading firefly page I wrote email and password, after I had error page and I decide restart container. And now I have issue 500.

This is container logs
[_FiryFlyIII_logs.txt](https://github.com/firefly-iii/firefly-iii/files/9794289/_FiryFlyIII_logs.txt)

**Comment by JC5** [comment]
It means the database isn't properly loaded with data. Try removing the underlying volumes and start both containers again.

**Reply by RobinGooze** [reply]
Thank you, all work!
